create_package.bat

```diff
@@ -0,0 +1,37 @@
+::
+:: Copyright (c) 2024 Intel Corporation
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http:::www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+::
+
+IF "%~1"=="" (
+    echo No argument provided. Using default opt path
+    set "output_user_root=opt"
+) ELSE (
+    echo Argument provided: Using install path %1
+    set "output_user_root=%1"
+)
+
```

Comment:
Add this PATH set to make msys tools available on laptops. set "PATH=C:\opt\msys64\usr\bin\;C:\opt;%PATH%;"

Reply:
msys64 is not going to be needed for the deployment. Setting PATH is also not required.